src/main/resources/mapper/PointMapper.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN"
+        "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
+<mapper namespace="org.example.dao.PointDAO">
+    <insert id="insertPoint" keyColumn="id" keyProperty="id" useGeneratedKeys="true">
+        INSERT INTO points (id, xCoordinate, yCoordinate) VALUES (#{id}, #{xCoordinate}, #{yCoordinate})
```

Comment:
I think these ones have to be names like in the database not like in java, i.e., x_coordinate not xCoordinate, etc.

Reply:
Sorry didn't notice that discrepancy!